examples/api/test/widgets/text_magnifier/text_magnifier.0_test.dart

```diff
@@ -96,16 +97,15 @@ void main() {
   }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS, TargetPlatform.android }));
 
 
-  for (final TextDirection textDirection in TextDirection.values) {
```

Comment:
Why not run this for both TextDirections any more?

Reply:
The LTR variation is exactly the same test as the one above.